packages/manager/modules/pci/src/projects/project/kubernetes/kubernetes.routing.js

```diff
@@ -53,6 +53,29 @@ export default /* @ngInject */ ($stateProvider) => {
         return promise;
       },
 
+      getKubeApiErrorId: /* @ngInject */ () => (error) => {
+        try {
```

Comment:
Why do you need `try…catch` here? I think you can get rid of it

Reply:
Relates to https://github.com/ovh/manager/pull/6138#discussion_r774609468